elftools/elf/notes.py

```diff
@@ -26,10 +26,13 @@ def iter_notes(elffile, offset, size):
         offset += nhdr_size
         elffile.stream.seek(offset)
         # n_namesz is 4-byte aligned.
```

Comment:
Nit: you probably want to move this comment (about alignment) into the `if` now, to be next to the `roundup`